pallets/anonymity-mining-claims/src/mock.rs

```diff
@@ -46,19 +48,19 @@ frame_support::construct_runtime!(
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
 		Balances: pallet_balances::{Pallet, Call, Storage, Event<T>},
-		HasherPallet: pallet_hasher::{Pallet, Call, Storage, Event<T>},
-		VAnchorVerifier: pallet_vanchor_verifier::{Pallet, Call, Storage, Event<T>},
-		LinkableTree: pallet_linkable_tree::{Pallet, Call, Storage, Event<T>},
-		MerkleTree: pallet_mt::{Pallet, Call, Storage, Event<T>},
+		// HasherPallet: pallet_hasher::{Pallet, Call, Storage, Event<T>},
+		HasherPallet: pallet_hasher::<Instance1>::{Pallet, Call, Storage, Event<T>},
+		VAnchorVerifier: pallet_vanchor_verifier::<Instance1>::{Pallet, Call, Storage, Event<T>},
+		ClaimsVerifier: pallet_claims_verifier::<Instance1>::{Pallet, Call, Storage, Event<T>},
+		LinkableTree: pallet_linkable_tree::<Instance1>::{Pallet, Call, Storage, Event<T>},
+		MerkleTree: pallet_mt::<Instance1>::{Pallet, Call, Storage, Event<T>},
 		Currencies: orml_currencies::{Pallet, Call},
 		Tokens: orml_tokens::{Pallet, Storage, Call, Event<T>},
 		AssetRegistry: pallet_asset_registry::{Pallet, Call, Storage, Event<T>},
-		VAnchor: pallet_vanchor::{Pallet, Call, Storage, Event<T>},
-		VAnchorHandler: pallet_vanchor_handler::{Pallet, Call, Storage, Event<T>},
-		SignatureBridge: pallet_signature_bridge::<Instance1>::{Pallet, Call, Storage, Event<T>},
+		VAnchor: pallet_vanchor::<Instance1>::{Pallet, Call, Storage, Event<T>},
 		TokenWrapper: pallet_token_wrapper::{Pallet, Call, Storage, Event<T>},
 		KeyStorage: pallet_key_storage::{Pallet, Call, Storage, Event<T>, Config<T>},
-		AnonymityMiningClaims: pallet_anonymity_mining_claims::{Pallet, Call, Storage, Event<T>, Config<T>}
+		AnonymityMiningClaims: pallet_anonymity_mining_claims::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>}
```

Comment:
I think we shouldn't make this pallet instance-able. That allows us to have multiple versions of the same pallet in the runtime, which does seem a bit overkill. The way to do this is to remove all `<I>` and instance trait extensions. Things like:
- `<T, I>` -> `<T>`
- `<T, I = ()>` -> `<T>`
- `<T: Config<I>, I: 'static = ()>` -> `<T: Config>`

Do book time with me if you want to pair and I can help you learn to do this quickly.

Reply:
Can a non-instanceable pallet take in an instanceable pallet in its Config trait. Because we take in a `VAnchor` type that represents the `APVAnchor` and `pallet_vanchor` is instanceable?

Reply:
I presume so, maybe @1xstj can chime in too.